src/code2vec.py

```diff
@@ -15,10 +19,11 @@ def code2vec(args):
     root, start_point = create_uast_source(args, session_name)
 
     res = start_point \
+        .link(Moder("func")) \
         .link(UastRow2Document()) \
         .link(UastDeserializer()) \
         .link(Uast2BagFeatures([UastPathsBagExtractor(args.max_length, args.max_width)])) \
-        .link(Collector()) \
+        .link(Vocabulary2Id(args.keep_freq, args.output)) \
```

Comment:
here add `root.sc` to initialize transformer with sparkContext

Reply:
I get this error:
```
  File "/home/hydra/projects/code2vec/src/../src/code2vec.py", line 26, in code2vec
    .link(Vocabulary2Id(root.sc, args.keep_freq, args.output)) \
AttributeError: 'Engine' object has no attribute 'sc'

```